synthesis.py

```diff
@@ -7,7 +7,7 @@
 options:
     --hparams=<parmas>                Hyper parameters [default: ].
     --preset=<json>                   Path of preset parameters (json).
-    --length=<T>                      Steps to generate [default: 32000].
```

Comment:
If we drop the default parameter then I don' think

https://github.com/r9y9/wavenet_vocoder/blob/2b557d4cbacef52bc3441fc4f2c54b1351ae9df4/synthesis.py#L146

will work, since `None` cannot be converted to `int`.